async-profiler-context/build.gradle

```diff
@@ -20,7 +20,9 @@ repositories {
 def asyncProfilerVersion = project.properties['async_profiler_version']
 def pyroscopeVersion = project.properties['pyroscope_version']
 dependencies {
-    api files("$buildDir/async-profiler/async-profiler.jar")
+// TODO
+    // api files("$buildDir/async-profiler/async-profiler.jar")
+    api files("/home/korniltsev/p/async-profiler/build/jar/async-profiler.jar")
```

Comment:
Is this on purpose?

Reply:
No, this is how i test locally, without publishing dependencies binaries to the maven central. This will be reverted once I publish jars